tmux/fragment_object_detection_debug.yml

```diff
@@ -52,27 +52,29 @@ tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
 # attach: false
 
 windows:
-  - datahub: ros2 run ptg_datahub datahub --ros-args
+  - datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
       -r __ns:=${ROS_NAMESPACE}
-      -p tcp_server_uri:=${HL2_IP}
-      -p det_3d_topic:=ObjectDetections3d
-      -p headset_pose_topic:=HeadsetPoseData
-  - detector: ros2 run py_pubsub object_detector --ros-args
+      -p ROS_IP:=${HL2_IP}
+  - image_converter: ros2 run angel_datahub ImageConverter --ros-args
       -r __ns:=${ROS_NAMESPACE}
-      -p image_topic:=PVFrames
+      -p topic_input_images:=PVFramesNV12
```

Comment:
Is it the unity-side stuff that now dictates this `PVFramesNV12` topic?

Reply:
Yes, that is correct.